components/account/AccountHeader.vue

```diff
@@ -103,7 +103,7 @@ const isNotifiedOnPost = $computed(() => !!relationship?.notifying)
             <AccountHandle :account="account" />
           </div>
         </div>
-        <div absolute top-18 inset-ie-0 flex gap-2 items-center>
+        <div absolute top-18 inset-ie-0 flex gap-2 justify-center>
```

Comment:
`items-center` here is correct. The issue is that the bell notification button is taller than the follow button. If the bell notification button size is reduced to match the follow button, then there will not be a pixel shift

Reply:
Thanks for the review!
I edited the commit and reverted the previous changes `justify-center` back to `item-center`  
I found out that the issue is resolved by changing the padding  of bell button from 0.5 rem to 0.4 rem `p2` to `p1.6` but I am getting type check error in CI check. I am still working on the issue.

Reply:
I think you needed to add `class="p1.6"` for this to work. I changed it back to `p2` though and made the icon `text-sm` so we have more coherent padding.